docs/fundamentals/startup.rst

```diff
@@ -1,8 +1,188 @@
-.. include:: /stub-topic.txt
+Application Startup
+===================
 
-|stub-icon| Application Startup
-===============================
+By `Steve Smith`_
 
-.. include:: /stub-notice.txt
+ASP.NET 5 provides complete control of how individual requests are handled by your application. The ``Startup`` class is the entry point to the application, setting up configuration and wiring up services the application will use. Developers configure a request pipeline in the ``Startup`` class that is used to handle all requests made to the application.
+
+In this article:
+	- `The Startup class`_
+	- `Understanding request delegates`_
+	- `Implementing middleware in its own class`_
+	- `Setting up your request pipeline`_
+	- `Run, Map, and Use`_
+	
+`Browse or download samples on GitHub <https://github.com/aspnet/Docs/tree/master/docs/fundamentals/startup/sample>`_.
+
+The Startup class
+-----------------
+
+In ASP.NET 5, the ``Startup`` class provides the entry point for an application. It's possible to have :doc:`environment-specific startup files and methods <environments>`, but regardless, one ``Startup`` class will serve as the entry point for the application. This ``Startup`` class can optionally accept dependencies in its constructor, such as an instance of ``IHostingEnvironment`` which can be used to customize the behavior of the class based on the environment in which it is running. Typically, the way an application will be configured is defined within its Startup class's constructor (learn more about :doc:`configuration`). The Startup class must define a ``Configure()`` method, and may optionally also define a ``ConfigureServices()`` method, which will be called by the server hosting the application when it is started.
+
+Configure()
+^^^^^^^^^^^
+
+The ``Configure()`` method is used to specify how the ASP.NET application will respond to individual HTTP requests. At its simplest, you can configure every request to receive the same response. However, most real-world applications require more functionality than this. The ASP.NET team recommends that more complex sets of pipeline configuration be encapsulated in extension methods on ``IApplicationBuilder``, to keep the ``Configure()`` method concise and easy to follow. In this example from the default web site template, you can see several extension methods are used to configure the pipeline with support for BrowserLink, error pages, static files, ASP.NET MVC, and Identity.
```

Comment:
I don't like the wording. This doesn't mention middleware at all. The recommendation is that each middleware is exposed via an extension method on IApplicationBuilder not arbitrary startup logic.